src/structures/NewsChannel.js

```diff
@@ -9,6 +9,7 @@ const { Error } = require('../errors');
  */
 class NewsChannel extends BaseGuildTextChannel {
   /**
+   * <info>This method returns itself. To get created webhook id, use `GuildChannelManager#addFolower`</info>
    * Adds the target to this channel's followers.
```

Comment:
> This method returns itself.

I'm not sure how to describe it, but this sounds confusing. This should be rephrased and placed below the description.

```suggestion
   * Adds the target to this channel's followers.
   * <info>If you need the created webhook id, use {@link GuildChannelManager#addFolower}.</info>
```